esm-samples/jsapi-angular-cli/README.md

```diff
@@ -4,14 +4,16 @@ This repo demonstrates how to use [@arcgis/core](https://www.npmjs.com/package/@
 
 ---
 ## Known issues
+* Upgrading Angular 15 automatically sets the TypeScript `target` to `ES2022` and `useDefineForClassFields` to `false`. The ArcGIS JS API requires that `useDefinedForClassFields` be set to `false` when using 
```

Comment:
This sounded a little confusing to me. Angular set this flag to `false`, and we also require it to be `false`? Maybe change the wording a bit to highlight that this change is in line with our requirements?

Reply:
Good point, I updated it and removed the Angular 15 reference.